plugins/kubernetes/app/views/samson_kubernetes/_project_form.html.erb

```diff
@@ -0,0 +1,4 @@
+<fieldset>
```

Comment:
so this is per project, not per stage?

Reply:
usually all stages deploy the same slow software, so would make sense to have it on the project 🤷‍♂

Reply:
I'm thinking more if say production or one pod has way more replicas than others, but yes i guess that's a different problem for us to solve a different way via "making progress" events or the like...

Reply:
can always crank this high as an escape hatch then :D